src/components/forms/Switch/Switch.stories.tsx

```diff
@@ -1,32 +1,145 @@
-import React from 'react';
 import { Meta, Story } from '@storybook/react/types-6-0';
+import React, { useState } from 'react';
 
+import { generateControl } from '../../../utils/tests/storybook';
+import { Stack } from '../../layout/Stack';
+import { Text } from '../../typography';
 import Switch from './Switch';
+import { SwitchSizes } from './Switch.enums';
 import { SwitchProps } from './Switch.types';
 
 export default {
   title: 'components/forms/Switch',
   component: Switch,
+  argTypes: {
+    maxWidth: { control: { type: 'number' } },
+    defaultChecked: { control: { type: 'boolean' } },
+    size: {
+      ...generateControl('select', SwitchSizes),
+    },
+  },
 } as Meta;
 
 export const playground: Story<SwitchProps> = (args) => <Switch {...args} />;
 playground.parameters = {
   chromatic: { disable: true },
 };
-playground.args = { label: 'Public', switchId: 'playground' };
+playground.args = {
+  label: 'Public',
+  switchId: 'playground',
+};
 playground.argTypes = {
```

Comment:
You can remove `argTypes` completely and move type's description to the `Switch.types.ts` file like in. https://github.com/securityscorecard/design-system/blob/alpha/src/components/Pill/Pill.types.ts#L11-L33

Reply:
Answered below. Please check it out.

Reply:
This is working with your suggestion. I will commit changes soon.  I have just copied what you did on Stepper